addons/website/views/snippets/snippets.xml

```diff
@@ -148,18 +148,35 @@
               data-unit="px" data-extra-class="rounded"/>
 </template>
 
+<template id="website.snippet_options_shadow">
+    <we-checkbox string="Shadow" data-select-class="shadow" data-name="shadow_active_opt" data-no-preview="true"/>
+    <we-multi class="" data-css-property="box-shadow" data-no-preview="true" data-dependencies="shadow_active_opt">
+        <we-colorpicker class="w-100 my-1" string="Color" data-select-style="" data-css-compatible=""/>
+        <we-row class="my-1">
+            <we-input string="X" data-select-style="" data-unit="px"/>
+            <we-input string="Y" data-select-style="" data-unit="px"/>
+        </we-row>
+        <we-row class="my-1">
+            <we-input string="Blur" data-select-style="" data-unit="px"/>
+        <we-input string="Spread" data-select-style="" data-unit="px"/>
+        </we-row>
+        <we-checkbox class="w-100 my-1" string="Inset" data-select-style="inset"/>
+    </we-multi>
+</template>
+
 <template id="snippet_options">
     <t t-call="web_editor.snippet_options"/>
 
     <!-- COLOR, BORDER, SHADOW | .s_three_columns | .s_comparisons -->
     <div data-selector=".s_three_columns .row > div, .s_comparisons .row > div"
-         data-target=".card">
+         data-target=".card"
+         data-js="shadow">
         <we-colorpicker string="Background Color"
             data-select-style="true"
             data-css-property="background-color"
             data-color-prefix="bg-"/>
         <t t-call="website.snippet_options_border"/>
-        <we-checkbox string="Shadow" data-select-class="shadow"/>
+        <t t-call="website.snippet_options_shadow"/>
```

Comment:
Something is not robust here: you use the "shadow" option which says "remove the box-shadow if you use selectClass with an empty value" but you use it on a complex set of widgets (border, background color, etc) which could use the selectClass method and thus remove the box shadow when you don't want to.

Either:
- Reinforce your JS option to say that it is when we remove the "shadow" class (`params.possibleValues`)
- Only use the shadow widgets in the shadow option.
- Both, ideally.

Reply:
I pushed all the changes, I'm just not sure about what you mean by
* Only use the shadow widgets in the shadow option.

Reply:
I meant you have a data-js="Shadow" which contains widgets which are not related at all to the shadow (background-color, border), but that can be ok I guess. I'll check your PR as soon as you mark the task green :+1:

Reply:
@qsm-odoo Ha yes I see, maybe the name should be different?

Reply:
Maybe, if you find a better one...